public/requests.js

```diff
@@ -0,0 +1,122 @@
+import md5_encode from './MD5.js'
+
+/**
+ *  добавление eventListener на кнопку поиска трека
+ */
+const search_input = document.querySelector('.header__search');
+search_input.addEventListener('keydown', function(e) {
+    if (e.keyCode === 13) {
+        send_request('GET',{method:'track.search',param:`&track=${search_input.value}`,track:search_input.value}).then(
+            res => {
+                if (res.hasOwnProperty('results')) {
```

Comment:
у undefined нет метода hasOwnProperty.
Лучше проверку написать так
```suggestion
                if (res?.hasOwnProperty('results')) {
```